apps/workers/db-check/worker.go

```diff
@@ -0,0 +1,157 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"math"
+	"time"
+
+	"github.com/jbsmith7741/uri"
+	"github.com/pcelvng/task"
+	"github.com/pcelvng/task-tools/slack"
+)
+
+type worker struct {
+	options
+	task.Meta
+	slack *slack.Slack
+
+	Table     string    `uri:"table" required:"true"`      // name of the table to query
+	DateField string    `uri:"date_field" required:"true"` // the date field to get row counts
+	Date1     time.Time `uri:"date1" format:"2006-01-02"`  // default is 24h back from today
+	Date2     time.Time `uri:"date2" format:"2006-01-02"`  // 2nd date to compare records against date1
+	Offset    string    `uri:"offset"`                     // duration back from date1 used to determine the value of date2 if needed
+	Percent   float64   `uri:"percent" default:"0.05"`     // allowed difference between date1 and date2 row counts
+}
+
+func (o *options) newWorker(info string) task.Worker {
+	w := &worker{
+		options: *o, // copy option values
+		Meta:    task.NewMeta(),
+		Date1:   time.Now().Add(-time.Hour * 24), // default date is yesterday
```

Comment:
It would be better to remove the default value and force the request to be explicit so that it is easier to rerun tasks and understand what is being compared.

Reply:
I can make this explicit, maybe that's better anyway so it's easier to read the workflow and know what is actually being checked.